catalog-info.yaml

```diff
@@ -39,8 +39,9 @@ spec:
     spec:
       repository: elastic/cloud-on-k8s
       provider_settings:
+        trigger_mode: "code"
+        build_pull_requests: true
         build_branches: true
-        build_pull_request_ready_for_review: true
```

Comment:
This consists of rebuilding pull requests when they are ready for review. I don't think we need it.